app/views/accessibility/show.html.erb

```diff
@@ -0,0 +1,101 @@
+<%= content_for :title, page_title(:accessibility) %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl"><%= t('page_titles.accessibility') %></h1>
+
+    <h2 class="govuk-heading-m">
+      How accessible this website is
+    </h2>
```

Comment:
If we're using `govuk-heading-m`, I believe these should be `h3` instead? Although I'm not sure if we should be going from `h1` to `h3` and if that's fine.

Reply:
Argh, you’re right. Have updated!

Reply:
@chubberlisk going from `h1` to `h3` is indeed a slight UX issue. Headings should always be progressively stair-cased in HTML 👍

The GOVUK Design System calls the heading classes `xl, l, m` for this reason, as opposed to `h1, h2, h3`, so they can be applied to whichever semantic header side makes sense given the context.